code/game/objects/items/blueprints.dm

```diff
@@ -47,7 +47,7 @@
 /obj/item/blueprints/interact()
 	var/area/A = get_area()
 	var/text = {"<HTML><head><title>[src]</title></head><BODY>
-<h2>[station_name()] blueprints</h2>
+<h2>[CONFIG_GET(string/server_name)] blueprints</h2>
```

Comment:
This would generate `TerraGov Marine Corps - Aliens vs Marines - Beginners welcome! blueprints`

Reply:
Wouldn't be that terrible tbh